### DISCUSSION TITLE
ESC-POS-.NET on Blazor WebAssembly

### DISCUSSION THREAD

**Discussion by arponcedeleon** [lukevp/ESC-POS-.NET#110] [General]
Hi, I am trying to use this ESC-POS-.NET in a WebAssembly Client Blazor project
but I am having problem when try to use it
first about the @using (need to do this to try to compile)
@using ESCPOS_NET
@using ESCPOS_NET.Extensions
@using ESCPOS_NET.DataValidation
@using ESCPOS_NET.Utilities
@using ESCPOS_NET.Emitters

but when try to run I have:
System.PlatformNotSupportedException: System.IO.Ports is currently only supported on Windows.
   at System.IO.Ports.SerialPort..ctor(String portName, Int32 baudRate)
   at ESCPOS_NET.SerialPrinter..ctor(String portName, Int32 baudRate)

questions:
Is it possible to use this ESC-POS-.NET on Blazor Client project (.NET Core 5) ??
do you have some suggestion how to to this??

thanks in advanced

**Comment by todosrc** [comment]
need Blazor server, imposible for Blazor client (web assembly)
